### PR TITLE
The Nextt Web Magazine clone: Responsive Design

### PR DESCRIPTION
**Pull request by atoledo-ec**
Contains:
-Flexbox, grids and float elements
-Used media query for responsive and different screen sizes
-Includes all elements on the original page

### CODE REVIEW

**Comment by umairarshadbutt** [comment]
##
## Pull Request under review by Umair Arshad (TSE)

File: styles.css
Comment (umairarshadbutt) [inline styles.css]: Remove commented css

File: styles.css
Comment (umairarshadbutt) [inline styles.css]: Remove commented CSS

File: styles.css
Comment (umairarshadbutt) [inline styles.css]: Remove commented CSS

File: index.html
Comment (umairarshadbutt) [inline index.html]: It should be an `article`

File: index.html
Comment (umairarshadbutt) [inline index.html]: It should be an `article`

File: index.html
Comment (umairarshadbutt) [inline index.html]: It should be an `article`

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @alejandrotoledoweb and @mke2111
Such a remarkable project you made :+1: ! Awesome README :100: !
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [x] Remove commented css from css file. ([Article for support](https://speckyboy.com/good-bad-css-practices/)) ( [screenshot](https://user-images.githubusercontent.com/29689273/93404399-8997b500-f8a3-11ea-8f88-fb12b8f6ca80.png) from Pathwright)
- [x] Use `article` tag properly
- [x] Margin of `main` tag effecting the navbar. Please fix the margin-top issue
### Navbar
- [x] Remove **GET TICKETS** button from small screens
- [x] Logo should not overflow on the sub-navbar


|[Source Page](https://thenextweb.com/)|[Clone Page](https://rawcdn.githack.com/mke2111/TNW-Clone/4ce29eedd4b895dd1c662492c70781430d1f67d0/index.html)|
|:---:|:---:|
|onav|c-nav478|


##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you stick at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @alejandrotoledoweb and @mke2111
Such a remarkable project you made :+1: ! Awesome README :100: !
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [x] Fix the Linter issues
- [x] Use `article` tag properly (As we discussed in call)


##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you stick at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:

## :wave: :wave: Hello!! Great effort on the project!

## Required Changes
### [Mandatory] :red_circle: 

- [x] Navbar its looking distorted on desktop screen size min 1024px
image

- [ ] Original website only has a 4 column lates news section
image

- [ ] Latest funding doesn't match original at 1024px
image
**and 1440px**
image

- [ ] Category section is not matching original at 1024px
image

- [ ] Footer has a white space on the right on mobile version

- [ ] Footer does not match the style of the original
image

- [ ] Latest funding show only 3 elements in mobile

- [ ] Latest funding has extra white space to the right
image

- [ ] 3 image section is lacking white space to the right
image


### [OPTIONAL]  :large_blue_circle: 

- [ ] The project should follow this structure. Having a directory for the css, the images and javascript files.
```
/
  css/
    style.css
  img/
    ... some images...
  js/
     ... some scripts...
  index.html
```

- [ ] Choose meaningful names for the CSS selectors. This improves readability and helps the programmer to keep track of fewer things in his mind.
**Actual**: 
```
.nn {
  ... some properties...
}
```
```
.navbar__menu {
  ... some properties...
}
```

- [ ] Use the media query browser inspector to compare the clone with the original. This will give a broad understanding of the project and will show if the clone looks as intended in different screen sizes.


## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Review by javitocor — CHANGES_REQUESTED** [review]
# STATUS CHANGES REQUIRED ♻️ 

Hello team!
Good job with the projects and the changes made! 👍 
Now the linters are green 🟢 

However, we still need to do some change before the approving the project and move forward ⏩ 
Kindly, check my comments below 👇  apply the changes and resubmit for a new code review

Good luck, guys! 😃 

- Kindly change the first section to 'latest news', now the title is 'latest deals'
- Navbar should be fixed or sticky
- Kindly update your live link before request for a code review

- At 1024px
   - Latest news section and latest deals sections should not have that much margin in the right, please make it equal to the original.
   - Latest fundings links(add info, see more...) must be inline, kindly modify it
   - The section after latest findings is overlapping one item with the other, please adjust the margins
- at 768px
   - latest fundings must have 1 item per row, kindly modify it
   - In the hard fork section, the first item, the latest section must be aligned left, please modify it
   - In the hard fork section, on the rest of the items, the latest section image must be aligned to the right
   - Please modify the footer so it takes all the width of the screen.

File: styles.css
Comment (javitocor) [inline styles.css]: maybe the problem with the footer comes from here, try to make it 100%

File: styles.css
Comment (javitocor) [inline styles.css]: maybe the problem with this section comes from here, try to modify the width to 100% and align the text to the left

**Review by javitocor — CHANGES_REQUESTED** [review]
#STATUS CHANGES REQUIRED ♻️ 

Team, good job so far with this project! 👍 
But still some code to fix to getting approved
Please, check my comments below 👇  and resubmit for a new code review when they are done 💪 

- The navbar is overlapping the content in medium and small devices, please fix it
- In small screens, the latest findings section must have a margin between items and should not have that big margin on the right side, please modify it and make it equal to the original.
- In small screens, the hard fork section, on the latest sections still have problems with the positioning, the image must be aligned right and the text must be aligned left in all items.
- The second latest news must be latest deals
- In the latest deals section kindly add the discount on the upper right corner

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__I must say, you've done an amazing job so far. However, there are some changes that are required to be fixed before this project can be approved. Please check my points below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

**_The main point of this project is to implement a Responsive Web Design. Therefore, we have to put emphasis on how it behaves on different screen sizes where the website is rendered_**

**At mobile screen sizes:**

- [x] You're having some alignment issues in the `Hard Fork` section.

Source page:


Your page:


- A similar case is observed in the `Apps`, `Gear`, `Tech`, `Creative`, `Podium`, `Insights`, `Launch`, and `Distract` sections too. Kindly fix it.

Source page:


Your page:


- [x] The `Latest Funding Rounds` section is having more space to the right than the left. Kindly fix this.

Source page:


Your page:



**Please make the required changes and submit another review request when you are done.**

* * *

**_You can reach out to me on twitter (@ola_lere), Github(@teekaytech) or slack (@taofeekolalere )_**
__Happy Coding :smiley:__

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @alejandrotoledoweb & @mke2111  :wave:

__I commend your efforts in implementing changes requested so far. The project is almost there, just some minor adjustments and we should be ready to go.__

**[ REQUIRED CHANGES ]**

_We have to make sure that everything is how they should be. It's a bit challenging to figure out all these changes at once because one thing is leading to another in this project._

**At 768px screen size**

- [ ] We are still having some whitespace issues in the `Latest Funding` section.

Source Page:


Your Page:


**At large screen size**

- [ ] The `Latest Funding` section is having some spaces different from the original page.
- Using `display: flex` for the latest fundings container might solve this problem.

Source page:


Your page:



**[ COMMENTS ]**
- I believe that most of these issues you are having can be traced back to the way you approach the implementation. 
- Using Grid/Flex for the containers of each section, most especially the ones we've been talking about, might really be of help. If they're properly used, Grid/Flex helps in justifying and aligning their contents the way we want them.

**Please make the required changes and submit another review request when you are done.**

* * *

**_You can reach out to me on twitter (@ola_lere), Github(@teekaytech) or slack (@taofeekolalere )_**
__Happy Coding :smiley:__

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @alejandrotoledoweb and @mke2111
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly work on the suggested changes  👇.
- [ ] The Latest Funding Rounds section is having more space to the right than the left. Kindly fix this.
##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you stick at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by javitocor — CHANGES_REQUESTED** [review]
# STATUS CHANGES REQUIRED ♻️ 

Team, good job with the implementation of the required changes! 👍 
But we still need to implement some more to get the project approved 💪 
Please, implement all the changes before for all screen sizes, make your project looks similar to the original, and then and only then request for a new code review.
Check my comments below 👇 

Good luck, guys! 😃 

- Large screens
   - latest fundings section must have the same width as latest news, latest deals and hard-fork section. Please fix it.
   - In the latest deals section, please add the yellow discount banner on the upper right side of the image.
- 1024px
   - Please remove the horizontal scrollbar
   - Latest foundation section width must have almost the full width of the screen, please centre it and give it the proper width.
   - Latest deals and latest news sections must have 3 items per row.
- at 768px:
  - please remove the horizontal scrollbar
   - Latest fundings section, the items must fit on the screen, now there is too much space in the right and left sides

**Comment by umairarshadbutt** [comment]
##
## Pull Request under review by Umair Arshad (TSE)

**Review by umairarshadbutt — APPROVED** [review]
# **Status: PROJECT APPROVED** :champagne:
Excellent job on this project guys. :fireworks: Keep it up.
##
### The [clone](https://rawcdn.githack.com/mke2111/TNW-Clone/98426bad9c0ea29af58e63f79c5dd2ea26ec093e/index.html) shows a good understanding of Building with Responsive Design. :100:
##
- The cloned page is rendering as it should and meets all the project specifications.
- The project has a descriptive README.md and is professionally done. Good job with that.
- Pull Request has a meaningful title and a short description.
- CSS is clean. No linter errors at this point and no bad practices used as well.
- The repo has followed all the guidelines there are to follow (GitHub flow etc.)
**All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.**
- Congratulations on the splendid job @alejandrotoledoweb and @mke2111! :+1: 
- You can now merge to master and submit the project completion form :rocket: . I wish you all the very best in your next tasks :1st_place_medal:
##
**If you have any questions, let me know on slack at any time. I will be happy to help.**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Pull request merged**
